css/templates/e-commerce/page.css

```diff
@@ -623,8 +638,12 @@ amp-selector [option][selected] {
 /** CHECKOUT */
 .commerce-checkout-steps {
   color: #cdcccd;
+  max-width: 350px;
```

Comment:
Fixed width for steps in the checkout flow. Not used elsewhere therefore not a css var.